src/helm/benchmark/scenarios/vision_language/fair_face_scenario.py

```diff
@@ -0,0 +1,136 @@
+from typing import List
+import os
+
+from datasets import load_dataset
+from tqdm import tqdm
+
+from helm.benchmark.scenarios.scenario import (
+    CORRECT_TAG,
+    VALID_SPLIT,
+    TRAIN_SPLIT,
+    Instance,
+    Input,
+    Output,
+    Reference,
+    Scenario,
+)
+from helm.common.media_object import MediaObject, MultimediaObject
+from helm.common.images_utils import generate_hash
+
+
+class FairFaceScenario(Scenario):
+    """
+    Identify the race, gender or age of a photo of a person.
+
+    @misc{kärkkäinen2019fairfacefaceattributedataset,
+      title={FairFace: Face Attribute Dataset for Balanced Race, Gender, and Age},
+      author={Kimmo Kärkkäinen and Jungseock Joo},
+      year={2019},
+      eprint={1908.04913},
+      archivePrefix={arXiv},
+      primaryClass={cs.CV},
+      url={https://arxiv.org/abs/1908.04913},
+    }
+
+    Paper: https://arxiv.org/abs/1908.04913
+    Website: https://huggingface.co/datasets/HuggingFaceM4/FairFace
+    """
+
+    HUGGINGFACE_DATASET_NAME: str = "HuggingFaceM4/FairFace"
+
+    RACE_ATTRIBUTE: str = "race"
+    GENDER_ATTRIBUTE: str = "gender"
+    AGE_ATTRIBUTE: str = "age"
+
+    QUESTION_TEMPLATE: str = "Identify the {attribute} of the person in the image."
```

Comment:
What's the purpose of including this classification task in VHELM? Which aspect does it measure?

Reply:
It's for fairness: we want to minimize the performance discrepancy across groups.

Reply:
Personally I think the premise of the original paper is questionable. Happy to discuss offline. I'm okay with including it for now.